### PR TITLE
[Docs][Connectors] Azure OpenAI and Hugging Face

### PR DESCRIPTION
**Pull request by igpetrov**
## Description

Added Azure OpenAI connector doc; also minor fixes to OpenAI/ChatGPT connector doc.

Hugging Face: coming soon in this PR.

## When should this change go live?

~`8.6.0-alpha1` release (the one in May, right?).~

Anytime.

- [ ] ~This change is not yet live and should not be merged until 2024-05-14 (apply `hold` label or convert to draft PR)?~
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by igpetrov** [comment]
Hey @christinaausley 

This one is ready for review but not for merge. No rush, raising in advance because it's quite big :)

File: docs/components/connectors/out-of-the-box-connectors/azure-open-ai.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+deployed [model](https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models) supports selected operation.
+
+### Completion
+
+Refer to the [official documentation page](https://learn.microsoft.com/en-us/azure/ai-services/openai/reference#completions)
+to learn more about correct usage, limitations, examples, and versions.
+
+### Chat completion
+
+Refer to the [official documentation page](https://learn.microsoft.com/en-us/azure/ai-services/openai/reference#chat-completions)
```
Comment (christinaausley) [inline docs/components/connectors/out-of-the-box-connectors/azure-open-ai.md:59]: Would it be better just to add an admonition saying "For completion details, refer to the Microsoft reference documentation."?

**Comment by christinaausley** [comment]
Moving out of `draft`, unless you have any additional large changes coming?

**Comment by igpetrov** [comment]
> Moving out of `draft`, unless you have any additional large changes coming?

Thanks a lot for reviewing, @christinaausley!

Not expecting major changes.

File: docs/components/connectors/out-of-the-box-connectors/azure-open-ai.md (hunk 1 above)
Comment (igpetrov) [reply docs/components/connectors/out-of-the-box-connectors/azure-open-ai.md]: Totally makes sense. Updated, please take a look if that looks better?

**Review by chillleader — DISMISSED** [review]

File: docs/components/connectors/out-of-the-box-connectors/azure-open-ai.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/connectors/out-of-the-box-connectors/azure-open-ai.md]: I condensed a little -- WDYT?

**Review by christinaausley — APPROVED** [review]

**Comment by igpetrov** [comment]
Hi @christinaausley 

I see that web-modeler has merged related changes, so this PR is good to merge anytime.

**Comment by igpetrov** [comment]
If it's okay with you, let's merge it now

**Pull request merged**